inscriptions/ISic000386.xml

```diff
@@ -185,21 +190,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141196" xml:lang="la">
+            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141196" xml:lang="la" resp="#JH">
                 <ab>
                     <lb n="1"/>
-                    <gap reason="lost" extent="unknown" unit="character"/><orig>NISTER</orig>
-                    <gap reason="lost" extent="unknown" unit="character"/>
+                        <gap reason="lost" extent="unknown" unit="character"/><orig>nister</orig>
+                        <gap reason="lost" extent="unknown" unit="character"/>
                     <lb n="2"/>
-                    <gap reason="lost" extent="unknown" unit="character"/><orig>DAER</orig>
+                        <gap reason="lost" extent="unknown" unit="character"/><orig>daer</orig>
                 </ab>
                 <!--This EDR text 141196 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp><app><note>Text of CIL</note></app></listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#JH">
```

Comment:
I'd simply delete this - one can't translate, and a repeat of the transcription doesn't add anything